parse-maintainer.py

```diff
@@ -104,5 +204,24 @@ def parse_issue(md):
     output_file = f"{username}.json"
     with open(output_file, 'w', encoding='utf-8') as f:
         f.write(json_output)
+        f.write("\n")  # Add trailing newline
     
     print(f"\nSaved to {output_file}", file=sys.stderr)
+    
+    # Validate against schema if requested
+    if validate_mode:
+        schema = load_schema()
+        if schema is None:
+            print("Schema file not found, skipping validation", file=sys.stderr)
```

Comment:
Since the schema file is available next to this file, will there be a case where it won't be found? I think this check would be redundant.